cvat/apps/documentation/installation.md

```diff
@@ -0,0 +1,308 @@
+    defining and running multi-container docker applications.
+
+    ```bash
+    sudo apt-get install -y python3-pip
+    sudo pip3 install docker-compose
+    ```
+
+-   Clone _CVAT_ source code from public
+    [GitHub repository](https://github.com/opencv/cvat).
+
+    ```bash
+    sudo apt-get install -y git
+    git clone https://github.com/opencv/cvat
+    cd cvat
+    ```
+
+-   Build docker images by default. It will take some time to download public
+    docker image ubuntu:16.04 and install all necessary ubuntu packages to run
+    CVAT server.
+
+    ```bash
+    docker-compose build
+    ```
+
+-   Run docker containers. It will take some time to download public docker
+    images like postgres:10.3-alpine, redis:4.0.5-alpine and create containers.
+
+    ```sh
+    docker-compose up -d
+    ```
+
+-   You can register a user but by default it will not have rights even to view
+    list of tasks. Thus you should create a superuser. The superuser can use
+    admin panel to assign correct groups to the user. Please use the command
+    below:
+
+    ```sh
+    docker exec -it cvat bash -ic 'python3 ~/manage.py createsuperuser'
+    ```
+    Choose login and password for your admin account. For more information
+    please read [Django documentation](https://docs.djangoproject.com/en/2.2/ref/django-admin/#createsuperuser).
+
+-   Google Chrome is the only browser which is supported by CVAT. You need to
+    install it as well. Type commands below in a terminal window:
+
+    ```sh
+    wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | sudo apt-key add -
+    sudo sh -c 'echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google-chrome.list'
+    sudo apt-get update
+    sudo apt-get install -y google-chrome-stable
+    ```
+
+-   Open the installed Google Chrome browser and go to [localhost:8080](http://localhost:8080).
+    Type your login/password for the superuser on the login page and press _Login_
+    button. Now you should be able to create a new annotation task. Please read
+    documentation about CVAT for more details.
+
+## Windows 10
+-   Download [Docker for Windows](https://download.docker.com/win/stable/Docker%20for%20Windows%20Installer.exe).
+    Double-click `Docker for Windows Installer` to run the installer. Complete
+    instructions can be found [here](https://docs.docker.com/docker-for-windows/install/). Note:
+    Docker Desktop requires Windows 10 Pro or Enterprise version 14393 to run.
+
+-   Download and install
+    [Git for Windows](https://github.com/git-for-windows/git/releases/download/v2.21.0.windows.1/Git-2.21.0-64-bit.exe).
+    When installing the package please keep all options by default.
+    More information about the package can be found [here](https://gitforwindows.org).
+
+-   Download and install [Google Chrome](https://www.google.com/chrome/). It is the only browser
+    which is supported by CVAT.
```

Comment:
If you have already installed Google Chrome we would recommend you update it.

Reply:
Don't want to complicate instructions.